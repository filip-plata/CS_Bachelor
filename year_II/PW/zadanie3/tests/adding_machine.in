4 1 4
1 x[0] = (x[1] + (x[2] * x[3]))
2 x[1] 17 x[2] -481
3 x[1] -1 x[2] -10 x[3] -2
4 x[1] -476
