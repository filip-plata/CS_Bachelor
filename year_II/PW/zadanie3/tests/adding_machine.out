1 P
2 F
3 P 19
4 F
